### ISSUE TITLE
Image Manipulation

### ISSUE THREAD

**Issue by lucentdan** [RPI-DATA/tutorials-intro#3]
Provide a function in PCA that flattens image.

**Comment by leondonglh** [comment]
Already solved this by using imread and flatten, but i just realize another question; when we dealt with single image, the flattens data will be 1 dimension such that we cannot plot it.  
Then for our eigenvalue plot, i will try to utilize a series of image just like the one in the " PCA15.ppt "

**Comment by leondonglh** [comment]
Already solved this by using imread and flatten, but i just realize another question; when we dealt with single image, the flattens data will be 1 dimension such that we cannot plot it.
Then for our eigenvalue plot, i will try to utilize a series of image just like the one in the " PCA15.ppt "

**Comment by leondonglh** [comment]
Already solved this by using imread and flatten, but i just realize another question; when we dealt with single image, the flattens data will be 1 dimension such that we cannot plot it.
Then for our eigenvalue plot, i will try to utilize a series of image just like the one in the " PCA15.ppt "

**Comment by lucentdan** [comment]
Yes - but we can store the image size and reshape the 1D array after we do PCA.
